ui/apps/platform/src/Containers/NetworkGraph/api/useModifyBaselineStatuses.ts

```diff
@@ -13,7 +11,7 @@ type Result = {
 type ModifyBaselineStatuses = {
     modifyBaselineStatuses: (
         flows: Flow[],
-        status: FlowStatus,
+        status: BaselineStatusType,
```

Comment:
nit: Also curious about the use of `BaselineStatusType` vs. something like `BaselineStatus`. The `Type` may not be necessary since we can deduce that it is a type.

Reply:
I think that name actually came from one of your [PR's](https://github.com/stackrox/stackrox/pull/4442) a while back. I'm not opposed to changing it, but it's being used in a lot of places.

Reply:
Ah, I see why I did that. The type is called `BaselineStatus,` so we couldn't call it that for the `status` field. But we should change it if we can find a better word for it.